flow/activities/flowable_core.go

```diff
@@ -11,7 +11,7 @@ import (
 
 	"github.com/jackc/pgx/v5"
 	"github.com/jackc/pgx/v5/pgxpool"
-	"github.com/yuin/gopher-lua"
+	lua "github.com/yuin/gopher-lua"
```

Comment:
you should fix your autoformat config settings to not do this

Reply:
@Amogh-Bharadwaj  can you share how you did this for my edification?

Reply:
i spent a good 30 mins unable to figure this out. Landing as is as this is a minor thing.